docker/config/local_dev.env

```diff
@@ -25,6 +25,9 @@ CRASHMOVER_CRASHPUBLISH_SUBSCRIPTION_NAME=local_dev_socorro_sub
 # Set Pub/Sub library to use emulator
 PUBSUB_EMULATOR_HOST=pubsub:5010
 
+# GCS settings
+STORAGE_EMULATOR_HOST=https://gcs-emulator:4443
```

Comment:
Is this really supposed to be `https`? With other emulators, I usually run `http` so we don't have to deal with certificates.

Reply:
I copied it from tecken, @biancadanforth could you answer this?

Reply:
to be clear, we can run this with http, per https://github.com/fsouza/fake-gcs-server/tree/main?tab=readme-ov-file#running-with-http, but tecken doesn't and i don't know why that choice was made

Reply:
The `fake-gcs-server` is fine with HTTP but the `google-cloud-storage` library is not IIRC. See also: https://mozilla.slack.com/archives/C060L2ZA0DT/p1703197794081399 .

Reply:
i got it working with http, so i'll do that here and update tecken